blockly/core/field_variable.js

```diff
@@ -124,7 +124,7 @@ Blockly.FieldVariable.prototype.setValue = function(newValue) {
  * @return {!Array.<string>} Array of variable names.
  * @this {!Blockly.FieldVariable}
  */
-Blockly.FieldVariable.dropdownCreate = function() {
```

Comment:
I'm tracking ardublockly, not blockly, so perhaps you can do it in your blockly branch?